pypeit/waveimage.py

```diff
@@ -71,9 +69,13 @@ def __init__(self, tslits_dict, tilts, wv_calib, spectrograph, maskslits,
 
         self.maskslits = maskslits
 
+        # For echelle order, primarily
```

Comment:
Should we just make slit_spat_pos an attribute of the tslits_dict?

Reply:
I'm fine either way.